tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+  "esm": true,
+  "compilerOptions": {
+    "module": "ESNext",
+    // "strict": true, //todo remove later
+    // "noImplicitAny": true, //todo remove later
+    // "strictNullChecks": true, //todo remove later
+    "noEmit": true,
+    "skipLibCheck": true,
+    "typeRoots": ["./node_modules/@types"],
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
```

Comment:
If we have `resolveJsonModule`, do we need to use the `require` method now seen in `cdk-cf.test.js`?

Reply:
No, we don't need to use the **require** method anymore.  
```
import cdkStack from './resources/cdk-ecs-cf.json' assert { type: 'json'}
``` 
But we should assert the type.